pkg/kubelet/apis/well_known_labels.go

```diff
@@ -23,8 +23,9 @@ const (
 
 	LabelInstanceType = "beta.kubernetes.io/instance-type"
 
-	LabelOS   = "beta.kubernetes.io/os"
-	LabelArch = "beta.kubernetes.io/arch"
+	LabelOS            = "beta.kubernetes.io/os"
+	LabelArch          = "beta.kubernetes.io/arch"
+	LabelKernelVersion = "beta.kubernetes.io/kernel-version"
```

Comment:
Why are we adding the `beta` prefix? What value does it provide? Given that renaming labels will break users, these labels will remain in beta forever.